xknx/knx/dpt_hvac_mode.py

```diff
@@ -37,6 +37,8 @@ def from_knx(cls, raw):
         elif raw[0] == 0x00:
             return HVACOperationMode.AUTO
 
+        return None
```

Comment:
Shouldn't we throw an exception here if we can't parse the KNX value? e.g. `raise CouldNotParseKNXIP("Could not parse HVACOperationMode")`